nixos/modules/services/networking/dhcpcd.nix

```diff
@@ -275,7 +276,7 @@ in
             RestrictSUIDSGID = true;
             SystemCallFilter = [
               "@system-service"
-              "~@aio" "~@chown" "~@keyring" "~@memlock"
+              "~@aio" "~@cpu-emulation" "~@debug" "~@keyring" "~@memlock" "~@mount" "~@obsolete" "~@privileged" "~@resources"
```

Comment:
@cpu-emulation is not part of system-service, so it's "~@cpu-emulation" is redundant.
Same for ~@debug, ~@obsolete, ~@privileged.

Reply:
Removed `~@cpu-emulation`, `~@debug` and `~@obsolete`.
Result, if remove `~@privileged` and add `~@chown`:
```
✗ SystemCallFilter=~@privileged                               System call allow list defined for service, and @privileged is included (e.g. capset is allowed)
```

Reply:
Ah, I missed it includes `@default`. Seems correct, to be sure I loaded the syscalls into python sets:
```
>>> system_service - cpu_emulation != systemservice
False
>>> system_service - debug != systemservice
False
>>> system_service - obsolete != systemservice
False
>>> system_service - chown != systemservice
True
>>> system_service - privileged != systemservice
True
```